.github/pull_request_template.md

```diff
@@ -9,3 +9,4 @@ Thank you for your pull request. Please provide a description below.
 - [ ] `.d.ts` file is updated
 - [ ] changelog is added, `npm run add-change`
 - [ ] run `bundle exec rake docs:api` and inspect `doc/latest/index.html` if documentation is changed
+- [ ] run `npm run integration` if integration test is changed
```

Comment:
Should there be any additional documentation on setting up the integration test if they're being added to the PR template?

Reply:
Yea I think I should.